=2.7

```diff
@@ -0,0 +1,8 @@
++ typescript@3.8.3
```

Comment:
This file `webcompat/=2.7` should not exist and is not part of the project. Probably a mistake, when you added or commit. 
Please remove it. :)